Makefile

```diff
@@ -62,21 +61,6 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-# Get latest git hash
-GIT_COMMIT_HASH=$(shell git rev-parse HEAD)
-
-# Get the local image registry required for clusterctl upgrade tests
-LOCAL_IMAGE_REGISTRY ?= localhost:5000
-
-ifeq (${MAKECMDGOALS},test-e2e-clusterctl-upgrade)
```

Comment:
have we tested test-e2e-clusterctl-upgrade to make sure we are not breaking it with this change

Reply:
Yes, you can see the tests that run and have passed in the status section at the end the PR.